### DISCUSSION TITLE
恕我直言，感觉越来越烂了。

### DISCUSSION THREAD

**Discussion by baagod** [immersive-translate/immersive-translate#2195] [Help]
### 概述 | Summary

一开始感觉很惊艳，到现在随便翻译一个网页，好多内容都是缺失的，一些元素会被直接屏蔽掉，根本用不了。

----
![582a6ec667775c0c7aa8b4368010d6a](https://github.com/user-attachments/assets/80436f76-c0b3-4636-9353-1eb5f71fb499)
![58ffd26f883ccda9462192ccb5f0ec6](https://github.com/user-attachments/assets/0f1a87a1-e94a-4754-9d9c-d4cc34ef0e44)
----

![2cb55652d1759870752c48852734964](https://github.com/user-attachments/assets/63453fee-5d9a-4663-a22f-afbc62a27e31)
![ac7f332653b0b204a13d8289341a296](https://github.com/user-attachments/assets/7aa6be4e-f77e-40c4-b2b1-9b4cc5126afc)
----

![618339cd4149ce3df9ddce7d2f68e9e](https://github.com/user-attachments/assets/d13df9f2-2481-44af-83f8-3837f648b9e8)
![73216dbf633a89199c8fb5ca0027146](https://github.com/user-attachments/assets/d5eeaf0a-9c9c-484f-9359-bf64c1258af7)

**Comment by theowenyoung** [comment]
请问可以附上具体的URL 和 缺失区域说明吗？第二个图看到是 标题缺失，其他的还没看到缺失的地方。
